windows-observ-lib/panels.libsonnet

```diff
@@ -0,0 +1,405 @@
+            table.standardOptions.withUnit('percent')
+          )
+          + fieldOverride.byName.withPropertiesFromOptions(
+            commonlib.panels.cpu.timeSeries.utilization.stylize()
+          ),
+        ])
+        + table.queryOptions.withTransformationsMixin(
+          [
+            {
+              id: 'joinByField',
+              options: {
+                byField: instanceLabel,
+                mode: 'outer',
+              },
+            },
+            {
+              id: 'filterFieldsByName',
+              options: {
+                include: {
+                  //' 1' - would only match first occurence of group label, so no duplicates
+                  pattern: std.join(' 1|', this.config.groupLabels) + ' 1|' + instanceLabel + '|product|^hostname$|Value.+',
+                },
+              },
+            },
+            {
+              id: 'organize',
+              options: {
+                excludeByName: {
+                  'Value #OS Info': true,
+                },
+                indexByName: {},
+                renameByName:
+                  {
+                    product: 'Product',
+                    [instanceLabel]: utils.toSentenceCase(instanceLabel),
+                    hostname: 'Hostname',
+                  }
+                  +
+                  // group labels are named as 'job 1' and so on.
+                  {
+                    [label + ' 1']: utils.toSentenceCase(label)
+                    for label in this.config.groupLabels
+                  },
+
+              },
+            },
+
+            {
+              id: 'renameByRegex',
+              options: {
+                regex: 'Value #(.*)',
+                renamePattern: '$1',
+              },
+            },
+          ]
+        ),
+      uptime: commonlib.panels.system.stat.uptime.new(targets=[t.uptime]),
+      systemContextSwitchesAndInterrupts:
+        commonlib.panels.all.timeSeries.base.new(
+          'Context switches/Interrupts',
+          targets=[
+            t.systemContextSwitches,
+            t.systemInterrupts,
+          ],
+          description=|||
+            Context switches occur when the operating system switches from running one process to another. Interrupts are signals sent to the CPU by external devices to request its attention.
+
+            A high number of context switches or interrupts can indicate that the system is overloaded or that there are problems with specific devices or processes.
+          |||
+        ),
+      systemExceptions:
+        commonlib.panels.all.timeSeries.base.new(
+          'System calls and exceptions',
+          targets=[
+            t.windowsSystemExceptions,
+            t.windowsSystemCalls,
+          ],
+        ),
+      systemThreads:
+        commonlib.panels.all.timeSeries.base.new(
+          'System threads',
+          targets=[
+            t.windowsSystemThreads,
+          ],
+        ),
```

Comment:
Same question as in `dashboards.libsonnet`.. If these are unused, should they persist?